src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -428,16 +430,20 @@ public void watchFiles(File buildFile, File outputDirectory, File testOutputDire
                     testSourceDirRegistered = false;
                 }
                 
-                // check if defaultConfigDirectory has been added and restart dev mode if it has
-                if (noConfigDir && this.defaultConfigDirectory.exists()){
-                    restartDevMode(executor);
-                }
-                
                 // check if configDirectory has been added
                 if (!configDirRegistered && this.configDirectory.exists()){
-                    restartDevMode(executor);
+                    configDirRegistered = true;
+                    debug("Configuration directory has been added: " + this.configDirectory);
+                    info("Unhandled change detected in pom.xml. Restart liberty:dev mode for it to take effect.");
```

Comment:
The message should be about configuration dir instead of pom.  Perhaps:
```
info("The server configuration directory " + configDirectory + " has been added. Restart liberty:dev mode for it to take effect.");
```